perl/bin/bb_vcf_to_ascat_cn.pl

```diff
@@ -0,0 +1,78 @@
+#!/software/perl-5.16.2/bin/perl
+
+#HACK do not commit!!
+##!/usr/bin/perl
+
+##########LICENCE##########
+# Copyright (c) 2016 Genome Research Ltd.
```

Comment:
Copyrights need updating to 2017.  I tend to do this on files as I edit them rather than a blanket update including unaltered files.  I'm happy to do this as part of the release branch if you want.